### DISCUSSION TITLE
Welcome to photocloud-frame-slideshow Discussions!

### DISCUSSION THREAD

**Discussion by mvysny** [mvysny/photocloud-frame-slideshow#171] [Announcements]
## Welcome

PhotoCloud Discussions are now live! Please feel free to ask any questions. The list of common questions is present at the [PhotoCloud FAQ page](https://www.android-photo-frame.eu/faq.html).

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by isaaias** [comment]
Hi!
I'm a fan of tour APP, I evento bought it and I realy line it.
You coul upgrade it to accept gifs, short videos or livre photos.

Tks
